code/modules/assembly/flash.dm

```diff
@@ -81,7 +80,6 @@
 
 /obj/item/assembly/flash/examine(mob/user)
 	. = ..()
-	. += "[bulb ? "The bulb looks like it can handle just about [bulb.charges_left] more uses.\nIt looks like you can cut out the flashbulb with a pair of wirecutters." : "The device has no bulb installed."]"
```

Comment:
Just take out the "The bulb looks like it can handle just about [bulb.charges_left] more uses.". Leave the rest as is.

Reply:
A better idea would be to have a tooltip that lets you know if the flash bulb is empty (0 charges), dim (1 or 2 charges) or bright (3 charges or more) so you can still tell if your flash runs out, but not meta the rev flashes.

Reply:
I don't consider it a better idea, I don't see a reason why would you need to check charges on your flash.

Reply:
Well, a flashbulb isn't like a regular bulb that works continuously. It's like on a camera, though you can't really inspect either of them.